lib/sdkGenHelper.js

```diff
@@ -156,9 +160,24 @@ function ensureTrailingComma (string) {
 function readPackageDependencies (sdkName, sdkDir) {
   var originFile = fs.readFileSync(sdkDir + '/Package.swift', 'utf8')
   var originMatches = originFile.match(/\.\bPackage\b.*/g)
+  var parsedDependencies = null
   if (originMatches) {
-    debug(`found ${originMatches.length} package dependencies in server SDK ${sdkName}`)
-    return originMatches.map(m => ensureTrailingComma(m.trim()))
+    parsedDependencies = []
+    originMatches.forEach(function (dependency) {
+      if (dependency.indexOf('minor') < 0) {
+        throw Error(chalk.red('SDKGEN Package dependency format is incompatible'))
+      }
+      dependency = dependency.replace('Package', 'package')
```

Comment:
A comment above this string processing would be helpful. Even just one example of expected input and output would be helpful.